source/includes/list-tables/system-env-vars.rst

```diff
@@ -0,0 +1,13 @@
+.. list-table::
```

Comment:
Is this file used anywhere? I don't see it.

Reply:
Good catch. Removed it.